probe_scraper/parsers/third_party/histogram_tools.py

```diff
@@ -517,11 +518,70 @@ def add_counter(context):
 
     return histograms
 
+
+def to_camel_case(property_name):
```

Comment:
Are there tests for this in moz-central?

Reply:
Yes, see https://phabricator.services.mozilla.com/D46111.

Reply:
Any luck running it locally?

Reply:
`python3 -m probe_scraper.runner --firefox-version 71 --dry-run` throws like this:

```
$ python3 -m probe_scraper.runner --firefox-version 71 --dry-run                                                                                                                                                                           
Unable to parse whitelist (/home/emilio/src/moz/probe-scraper/probe_scraper/parsers/third_party/histogram-whitelists.json). Assuming all histograms are acceptable.

nightly - extracted version data:
  {'node': 'fd2934cca1ae7b492f29a4d240915aa9ec5b4977', 'version': 71}
  {'node': '9596d7f4a7457bccc78cadf9c39bcc9c4b5b97f8', 'version': 71}

nightly - loading files:
  from: {'node': 'fd2934cca1ae7b492f29a4d240915aa9ec5b4977', 'version': 71}
  from: {'node': '9596d7f4a7457bccc78cadf9c39bcc9c4b5b97f8', 'version': 71}
Traceback (most recent call last):
  File "/usr/lib64/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib64/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/emilio/src/moz/probe-scraper/probe_scraper/runner.py", line 343, in <module>
    args.firefox_channel)
  File "/home/emilio/src/moz/probe-scraper/probe_scraper/runner.py", line 282, in main
    min_firefox_version, firefox_channel)
  File "/home/emilio/src/moz/probe-scraper/probe_scraper/runner.py", line 164, in load_moz_central_probes
    channels=channels)
  File "/home/emilio/src/moz/probe-scraper/probe_scraper/scrapers/moz_central_scraper.py", line 272, in scrape
    versions = extract_tag_data(tags, channel, min_fx_version, max_fx_version)
  File "/home/emilio/src/moz/probe-scraper/probe_scraper/scrapers/moz_central_scraper.py", line 159, in extract_tag_data
    if (tip_node_id != results[-1]["node"] and
IndexError: list index out of range
```

which doesn't seem related to my changes. Anything I'm doing wrong?

Reply:
Ah, got it, `--firefox-channel nightly`.